.github/workflows/test.yaml

```
@@ -51,7 +51,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes: [v1.24, v1.20]
+        kubernetes: [v1.24, v1.21]
```

```suggestion
        kubernetes: [latest, v1.21]
```
We had to pin this at one point, but probably can go back to `latest` now (which should be 1.25)